<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-QJ8FUFK" failures="1" tests="10" name="Surefire test" time="16.408" errors="0" timestamp="2023-10-11T16:35:28 ICT">
  <testcase classname="api.test.DataDrivenTest" name="testPostUser" time="2.916"/>
  <testcase classname="api.test.DataDrivenTest" name="testPostUser" time="1.246"/>
  <testcase classname="api.test.DataDrivenTest" name="testPostUser" time="1.24"/>
  <testcase classname="api.test.DataDrivenTest" name="testDeletUserByName" time="1.248"/>
  <testcase classname="api.test.DataDrivenTest" name="testDeletUserByName" time="1.277"/>
  <testcase classname="api.test.DataDrivenTest" name="testDeletUserByName" time="1.208"/>
  <testcase classname="api.test.UserTest" name="testPostUser" time="1.321">
    <failure type="java.lang.AssertionError" message="expected [204] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [204] but found [200]
at api.test.UserTest.testPostUser(UserTest.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 39 stack frames]]>
    </failure>
  </testcase> <!-- testPostUser -->
  <testcase classname="api.test.UserTest" name="testGetUserByName" time="1.347"/>
  <testcase classname="api.test.UserTest" name="testUpdateUserByName" time="1.204"/>
  <testcase classname="api.test.UserTest" name="testDeleteUserByName" time="1.345"/>
</testsuite> <!-- Surefire test -->
